napari/utils/transforms/_tests/test_transforms.py

```diff
@@ -18,6 +18,26 @@ def test_scale_translate(Transform):
     npt.assert_allclose(new_coord, target_coord)
 
 
+@pytest.mark.parametrize('Transform', [Affine, CompositeAffine])
+def test_affine_is_diagonal(Transform):
+    transform = Transform(scale=[2, 3], translate=[8, -5], name='st')
+    assert transform.is_diagonal
+    transform.rotate = 5.0
+    assert not transform.is_diagonal
+    # Rotation back to 0.0 will result in tiny non-zero off-diagonal values.
+    # is_diagonal assumes values below 1e-8 are equivalent to 0.
+    transform.rotate = 0.0
+    assert transform.is_diagonal
+
+
+def test_diagonal_scale_setter():
+    # diagonal matrices are also considered a permutation
```

Comment:
Is this comment left over from the larger PR? Permutations don't seem related to this test as is.